posthog/management/commands/start_temporal_worker.py

```diff
@@ -63,6 +66,9 @@ def handle(self, *args, **options):
             options["client_key"] = "--SECRET--"
         logging.info(f"Starting Temporal Worker with options: {options}")
 
+        port = int(os.environ.get("PROMETHEUS_METRICS_EXPORT_PORT", 8001))
+        start_http_server(port=port)
```

Comment:
I tested this out locally and it works as expected (it runs the metrics server in another thread, so it shouldn't mess with async or anything else). I'll need to wire up the scrapers once we deploy this, of course.

Reply:
Even if running on a separate thread, if the blocking IO doesn't release the GIL we would still be blocking the event loop. But, if it's working in tests then the IO should be playing nice and releasing the GIL 🤝